app/controllers/concerns/idv/verify_info_concern.rb

```diff
@@ -330,6 +330,7 @@ def add_proofing_costs(results)
           add_cost(:lexis_nexis_resolution, transaction_id: hash[:transaction_id])
         elsif stage == :state_id
           next if hash[:vendor_name] == 'UnsupportedJurisdiction'
```

Comment:
I think this is erroneous. AAMVA appears to bill us for anything that doesn't return an exception, whether or not a match is found.

Reply:
if we think it's erroneous, let's remove it

Reply:
Agreed, but wanted to make sure there was buy-in on doing so.